scripts/superglue-baselines.sh

```diff
@@ -22,7 +22,7 @@ function multirc() {
 }
 
 function rte() {
-    python main.py --config config/superglue-bert.conf --overrides "random_seed = ${seed}, cuda = ${gpuid}, run_name = rte, pretrain_tasks = \"rte-superglue\", target_tasks = \"rte-superglue,superglue-diagnostic\", do_pretrain = 1, do_target_task_training = 0, do_full_eval = 1, batch_size = 4, val_interval = 625"
+    python main.py --config config/superglue-bert.conf --overrides "random_seed = ${seed}, cuda = ${gpuid}, run_name = rte, pretrain_tasks = \"rte-superglue\", target_tasks = \"rte-superglue,superglue-diagnostic,winograd-diagnostic\", do_pretrain = 1, do_target_task_training = 0, do_full_eval = 1, batch_size = 4, val_interval = 625"
```

Comment:
We can't use this name for the task: It's not in Winograd format and it's not associated with the Winograd task in jiant. How about superglue_bias_diagnostic?

Reply:
Winogender works, too.

Reply:
Or winogender-diagnostic.